Gemfile.lock

```diff
@@ -140,6 +142,10 @@ GEM
     rb-inotify (0.10.1)
       ffi (~> 1.0)
     regexp_parser (1.7.1)
+    rspec (3.9.0)
+      rspec-core (~> 3.9.0)
+      rspec-expectations (~> 3.9.0)
+      rspec-mocks (~> 3.9.0)
```

Comment:
What is this for?

Reply:
Probably something that the NyanCat installed.  Re: @AngelaGuardia after I asked her about it over zoom.